lib/pure/unicode.nim

```diff
@@ -1602,6 +1605,25 @@ iterator runes*(s: string): Rune =
     fastRuneAt(s, i, result, true)
     yield result
 
+when not defined(nimscript):
+  iterator runes*(s: Stream): Rune =
+    ## Iterates over any unicode character of
+    ## the stream ``s`` returning runes. Raises `EIO`
+    ## if an error occurred.
+    var buff = newString(4).TaintedString
```

Comment:
The ``.TaintedString`` and ``string`` conversions make the code less readable.

Reply:
I only included that coz it's everywhere in the stream module. I've read those are for helping in debugging, but I don't even know what it's good for really.

Reply:
Well, change it.